tests/test_backed_sparse.py

```diff
@@ -318,12 +345,69 @@ def test_indptr_cache(
     assert store.get_access_count("X/indptr") == 2
 
 
+Kind = Literal["slice", "int", "array", "mask"]
+
+
+def mk_idx_kind(idx: Sequence[int], *, kind: Kind, l: int) -> Idx | None:
```

Comment:
Maybe a doc string here?  Just a description would be good to help reason about what is going on

Reply:
Sorry, consecutive integers, not start-stop pair